scripts/stac/imagery/collection.py

```diff
@@ -210,94 +210,70 @@ def write_to(self, destination: str) -> None:
 
     def _title(self) -> str:
         """Generates the title for imagery and elevation datasets.
-        Satellite Imagery / Urban Aerial Photos / Rural Aerial Photos:
-          [geographic_description / Region if no geographic_description specified] [GSD] [?Event Name] [Data Sub-Type]
-          ([Year(s)]) [?- Preview]
+        Satellite Imagery / Urban Aerial Photos / Rural Aerial Photos / Scanned Aerial Photos:
+          https://github.com/linz/imagery/blob/master/docs/naming.md
         DEM / DSM:
-          [geographic_description / Region if no geographic_description specified] [?- Event Name] LiDAR [GSD]
-          [Data Sub-Type] ([Year(s)]) [?- Preview]
-        If Historic Survey Number:
-          [geographic_description / Region if no geographic_description specified] [GSD] [Survey Number] ([Year(s)])
-          [?- Preview]
-
+          https://github.com/linz/elevation/blob/master/docs/naming.md
         Returns:
             Dataset Title
         """
         # format optional metadata
         geographic_description = self.metadata.get("geographic_description")
         historic_survey_number = self.metadata.get("historic_survey_number")
-        event = self.metadata.get("event_name")
 
         # format date for metadata
+        date = f"{self.metadata['start_datetime'].year}-{self.metadata['end_datetime'].year}"
         if self.metadata["start_datetime"].year == self.metadata["end_datetime"].year:
             date = str(self.metadata["start_datetime"].year)
```

Comment:
```python
if something:
    foo = other
else:
    foo = default
```

vs.

```python
foo = default
if something:
    foo = other
```

isn't always clear cut:

- The second variant is shortest.
- The first variant is less computationally costly, because we never compute a value and then throw it away.
- The first variant is (IMO) clearer because the values are at the same indentation level.
- The first variant avoids variable re-use, which makes debugging easier.
- All that said, if `default` has a side-effect the two variants aren't equivalent.

On balance, I'd consider the advantages of the first variant to usually outweigh the shorter second variant, as long as the side-effect isn't relevant.